setup.py

```diff
@@ -54,6 +54,19 @@ def get_version():
     ver = version.rsplit(" ", 1)[-1][1:]
     return ver
 
+classifiers = [
+    'Development Status :: Aplha',
+    'Topic :: Software Development :: Build Tools',
+    'Intended Audience :: Science/Research',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: Apache 2.0 License',
```

Comment:
according to https://pypi.org/classifiers/
```suggestion
    'License :: OSI Approved :: Apache Software License',
```

Reply:
Done